src/Form/UserAccount/UserProfileType.php

```diff
@@ -101,7 +93,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         ;
     }
 
-    /** @return FormInterface|void */
+    /** @return FormInterface|null */
     public function onPasswordSubmit(FormEvent $event): mixed
```

Comment:
Let's use a real type hint instead of this PHPDoc :/ 

BTW, what's the point to return the form? the return value is never used. let's never return something

Reply:
Actually the value is used by the validator. I can't exactly remember where and how, it would require digging, but I'm 100% sure that if I remove the returns and the user submits an invalid form, it breaks.